### DISCUSSION TITLE
StreamPipes is going to be a mentor community for OSPP 2023

### DISCUSSION THREAD

**Discussion by liuxiaocs7** [apache/streampipes#1460] [General]
Hello, community.

When I browsed [this website](https://summer-ospp.ac.cn/org/orglist), I noticed that the streampipes community will also participate in OSPP2023(An open source event in china similar to GSOC). As a Chinese student, it is very exciting and looking forward to the task.

![image](https://user-images.githubusercontent.com/42756849/228302174-0fc5ad31-0909-4765-841e-758f12b2692a.png)

**Comment by bossenti** [comment]
@liuxiaocs7 great that you already discovered us 🚀 
It's the first time we take part in the OSPP and are already excited to see the applications 🤩
